index/es7-persistence/src/main/java/com/netflix/conductor/es7/dao/index/ElasticSearchRestDAOV7.java

```diff
@@ -1341,102 +1341,4 @@ BulkRequestWrapper getBulkRequest() {
             this.bulkRequest = new BulkRequestWrapper(bulkRequest);
         }
     }
-
-    @Override
-    public void removeTask(String workflowId, String taskId) {
-        try {
-            long startTime = Instant.now().toEpochMilli();
-            String docType = TASK_DOC_TYPE;
```

Comment:
Hi @bjpirt , Looks like this is the correct method. Ideally, we should be removing the other one. Can you please verify?

Reply:
I thought document types had been deprecated (as per https://github.com/Netflix/conductor-community/pull/170) and the calls in the functions I removed to `UpdateRequest` were displaying in my IDE as deprecated there, hence getting rid of these functions.

Reply:
Hi @james-deee and @bjpirt , Let me verify this.

Reply:
Hi @james-deee, @bjpirt , Changes looks fine. LGTM